bin/functions.sh

```diff
@@ -57,7 +72,11 @@ gs_delete_file() {
 #            2. target s3 url (s3://...)
 s3_copy_file() {
 	if [ $# -ne 2 ]; then return 255; fi
-	${AWSCLI} ${AWSCLIOPT} ${AWSCLI_COPY_OPT} $1 $2
+		if [ -z "$AWSCLI_ENDPOINT_OPT" ]
+		then
+			${AWSCLI} ${AWSCLIOPT} ${AWSCLI_COPY_OPT} $1 $2
+		else
+			${AWSCLI} --endpoint-url ${AWSCLI_ENDPOINT_OPT} ${AWSCLIOPT} ${AWSCLI_COPY_OPT} $1 $2
```

Comment:
There is no "fi" on line 80 and it is a syntax error.

```suggestion
  ${AWSCLI} ${AWSCLI_ENDPOINT_OPT} ${AWSCLIOPT} ${AWSCLI_COPY_OPT} $1 $2
```